src/js/controller/controller.js

```diff
@@ -422,7 +422,7 @@ Object.assign(Controller.prototype, {
 
             if (_model.get('state') === STATE_COMPLETE) {
                 _stop(true);
-                _setItem(0);
+                _this.setItemIndex(0);
```

Comment:
There are five calls to `_setItem` in the controller. Should we not make all calls go through the instance `_this.setItemIndex`?

Reply:
They are the same, as line 810 assigns `this.setItemIndex = _setItem`. For clarity, we can probably always use the `this` variant.